src/WizardsWardrobeGui.lua

```diff
@@ -48,6 +48,47 @@ function WWG.Init()
 	zo_callLater( function() WWG.OnWindowResize( "stop" ) end, 250 )
 end
 
+function WWG.SetupExitWarning()
+	local menu = WINDOW_MANAGER:GetControlByName("ZO_GameMenu_InGame").gameMenu
+	local logoutControl
+	local quitControl
+
+	for i, children in ipairs(menu.navigationTree.rootNode.children) do
+		if children.control:GetText() == string.upper(GetString(SI_GAME_MENU_LOGOUT)) then
+			logoutControl = children.control
+		elseif children.control:GetText() == string.upper(GetString(SI_GAME_MENU_QUIT)) then
+			quitControl = children.control
+		end
+	end
+
+	if not WW.settings.showExitWarnings then
```

Comment:
If the warnings are disabled, a lot of the rest of the function can be skipped too. No point iterating gear if warnings are diabled

Reply:
Ah, the error was because you're referencing controls that don't exist yet